Funcky.Test/Funcky.Test.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
-        <TargetFrameworks>net8.0;net7.0;net6.0;net5.0;netcoreapp3.1</TargetFrameworks>
+        <TargetFrameworks>net8.0;net7.0;net6.0</TargetFrameworks>
```

Comment:
What is our policy here?

Reply:
I don't think we have one... Up until now we just kept testing against all the old TFMs.

I removed the obsolete ones for two reasons:
* netcoreapp3.1 causes trouble on CI (and I was to lazy to investigate)
* netcoreapp3.1 and net5.0 are no longer supported by the latest versions of xUnit

Downside is that we don't test against TFMs that we target in `Funcky`.
But then again: Who is using these anyways?

Reply:
OK, l close this and add a task to adding a SupportPolicy.md